ui/shared/utils/constants.js

```diff
@@ -108,6 +109,12 @@ export const FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP = FAMILY_SUCCESS_STORY_TYP
   return { [tag.value]: tag, ...acc }
 }, {})
 
+export const FAMILY_SUCCESS_STORY_TYPE_TAG = tag =>
```

Comment:
this is a confusing variable name, I would expect something with this name to be the string value of the field key or something. For a function, you should use camel case and make it describe what it does, so maybe something like `successStoryTypeDisplay`?

Reply:
Also, I think this should be the default display for the tag field view, which is colored labels right?